apache/config.inc.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
+define('SSL_DIR', '/etc/phpmyadmin/ssl');
```

Comment:
you could add an ENV in the Dockerfile and it would be available in $_ENV[ instead

Reply:
will be done in minutes

Reply:
done